resources/resources.go

```diff
@@ -36,20 +36,17 @@ func InstallPostgres(resourceDir, dataDir string) (string, error) {
 		return "", fmt.Errorf("OS %q is not supported", runtime.GOOS)
 	}
 
-	// Skip installation if installed already.
 	pgBinDir := path.Join(resourceDir, version)
 	_, err := os.Stat(pgBinDir)
 	if err != nil && !os.IsNotExist(err) {
-		return "", fmt.Errorf("failed to check binary path %q, error: %w", pgBinDir, err)
+		return "", fmt.Errorf("failed to check binary directory path %q, error: %w", pgBinDir, err)
```

Comment:
I am still thinking to put both install pg and initDB inside a single method. like BootstrapPostgres.

The reason is initDB putting outside is a bit confusing as it might mean to also run the migration scripts.

Reply:
Done. Make sense.